src/CMakeLists.txt

```diff
@@ -323,6 +323,8 @@ elseif ( WIN32 )
   set_target_properties ( libfluidsynth
     PROPERTIES
       PUBLIC_HEADER "${public_HEADERS}"
+      ARCHIVE_OUTPUT_NAME "libfluidsynth-${LIB_VERSION_CURRENT}"
+      PREFIX "lib"
```

Comment:
This has the effect of naming the import library in MinGW as:

`-- Installing: D:/a/fluidsynth/fluidsynth/fluidsynth_install/lib/liblibfluidsynth-3.dll.a`

https://github.com/FluidSynth/fluidsynth/actions/runs/14477269150/job/40605816818

Reply:
So the `PREFIX` applies to
* import lib + static lib of MinGW, and
* the DLL itself obviously.

But it does not apply to the import lib MSVC, which is [named correctly](https://dev.azure.com/tommbrt/tommbrt/_build/results?buildId=12360&view=logs&j=e765dd6c-0c81-5220-d856-9f489fdad6be&t=61587a34-96e9-564a-c9f3-8ebf5db1c48a&l=132). Very strange. I'll add more checks. The documentation for [PREFIX](https://cmake.org/cmake/help/v3.31/prop_tgt/PREFIX.html) is also pretty much non-existent. Let's see if we can avoid using it, at least on Windows.